examples/kitchen-sink/dashboard.py

```diff
@@ -25,5 +25,7 @@ class Project(FlowProject):
     if "dashboard" not in project.document:
```

Comment:
This line is a relic from when I stored some dashboard information in the project document. It can be removed.